crates/rover-client/src/error.rs

```diff
@@ -249,7 +249,7 @@ fn check_workflow_error_msg(check_response: &CheckWorkflowResponse) -> String {
         },
         if let Some(lint_response) = &check_response.maybe_lint_response {
```

Comment:
I don't want to add too much scope, but do we want to also rename all the variables and classes as well from "lint" to "linter"?

Reply:
I did that but the diff was pretty significant so i reverted my changes /:

If we still want that i can get those changes in but it might make reviewing more tedious.